### PR TITLE
Rtracklayer

### PR DESCRIPTION
**Pull request by tina-r**
* [ ] I have read the guidelines above.
* [ ] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/bioconductor-rtracklayer/bioconductor-rtracklayer/build.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# R refuses to build packages that mark themselves as
+# "Priority: Recommended"
+mv DESCRIPTION DESCRIPTION.old
+grep -v '^Priority: ' DESCRIPTION.old > DESCRIPTION
+#
+$R CMD INSTALL --build .
+#
+# # Add more build steps here, if they are necessary.
+#
+# See
```
Comment (bgruening) [inline recipes/bioconductor-rtracklayer/bioconductor-rtracklayer/build.sh]: Can you please remove the comments.

File: recipes/bioconductor-rtracklayer/bioconductor-rtracklayer/build.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
+
```
Comment (bgruening) [inline recipes/bioconductor-rtracklayer/bioconductor-rtracklayer/build.sh]: first empty line should also be removed.

**Comment by bgruening** [comment]
@tina-r you wanted to get a newer version?

**Comment by johanneskoester** [comment]
Closing for now. Please simply update the existing recipe.

**Pull request closed without merging**